default/python/AI/empire/survey_universe.py

```diff
@@ -126,12 +121,9 @@ def survey_universe():
                 set_colonies_is_under_treat()
 
     survey_universe_lock.unlock()
-    # main lock must be unlocked before get_pilot_ratings is called
-    rating_list = sorted(get_pilot_ratings().values(), reverse=True)
-    summarize_pilot_ratings(rating_list)
-    survey_universe_lock2.unlock()
     survey_timer.stop()
 
+    # lock must be released before get_pilot_ratings is called
```

Comment:
Not sure about this comment.

Reply:
Actually it is nonsense now, I should have changed it with my second commit.